src/compas/data/schemas/box.json

```diff
@@ -0,0 +1,29 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "box.json",
+    "$compas": "1.6.3",
```

Comment:
shouldn't all of these be changed to `"1.7.1"`?

Reply:
yes indeed